### PR TITLE
adding attributes for cloud front distribution

### PR DESCRIPTION
**Pull request by Rohit1509**
Signed-off-by: Rohit Joshi <rohit.prasad.joshi@sap.com>

### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec AWS](https://github.com/inspec/inspec-aws/CONTRIBUTING.md) document before submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [ ] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [x] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: libraries/aws_cloudfront_distribution.rb [hunk 1]
```diff
@@ -86,6 +91,10 @@ def exists?
     !@distribution_arn.nil? && @distribution_arn.start_with?('arn')
   end
 
+  def access_logging_enabled?
+    config.logging.enabled == false
```
Comment (sathish-progress) [inline libraries/aws_cloudfront_distribution.rb:95]:
```suggestion
    !!config.logging.enabled
```

File: libraries/aws_cloudfront_distribution.rb [hunk 2]
```diff
@@ -60,17 +62,20 @@ def initialize(opts = {})
 
     @custom_origin_protocol_policies = []
     @custom_origin_ssl_protocols = []
+    @s3_origin_access = []
     @s3_origin_configs = false
     config.origins.items.each do |origin|
       if origin[:s3_origin_config]
         @s3_origin_configs = true
+        @s3_origin_access += [origin[:s3_origin_config][:origin_access_identity]]
```
Comment (sathish-progress) [inline libraries/aws_cloudfront_distribution.rb:70]:
```suggestion
        @s3_origin_access << origin[:s3_origin_config][:origin_access_identity]
```

File: libraries/aws_cloudfront_distribution.rb (hunk 1 above)
Comment (Rohit1509) [reply libraries/aws_cloudfront_distribution.rb]: fixed

File: libraries/aws_cloudfront_distribution.rb (hunk 2 above)
Comment (Rohit1509) [reply libraries/aws_cloudfront_distribution.rb]: fixed

**Review by sathish-progress — APPROVED** [review]
LGTM

**Pull request merged**